docs/project/reward-mechanism.mdx

```diff
@@ -5,13 +5,18 @@ sidebar_label: Reward Mechanism
 slug: /reward-mechanism
 ---
 
-Rewarding the members of our network for sharing their local weather data is a crucial part of our system (**currently on testnet**).
+## Introduction
+
+:::important
+The Reward Mechanism is currently at **version 1.0**. This page serves as both an informational resource about its current status and a glimpse into its future.
+:::
+
+- Also include a short overview of the timeline here
```

Comment:
`also include a short overview of the timeline here`
This is a placeholder :stuck_out_tongue: 
Please include an intro outlining:
- What is the reward mechanism
- What purpose does it serve
- Something along the lines of: 
At present the Reward Mechanism stands at version 1.0. This version of the RM allocates rewards by evaluating only basic station data adequacy (whether a weather station sent enough data for the period or not). 
In RM v1.5 the QoD v1.0 and PoL v1.0 mechanisms are integrated into the pipeline of rewards. With QoD the weather data quality is evaluated using more advanced checks, while PoL verifies the location data provided by the weather station.
In RM v2.0 ...
[pls expand the above as you see fit]

Following is an in-depth view of the inner workings of the current RM version, as well as future milestones.

Reply:
I understood the placeholder part, no worries.